frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -68,20 +56,22 @@ const Header: React.FC = () => {
 
   return (
     <>
-      <AppBar position="static" color="secondary">
+      <AppBar position="static" color="secondary" elevation={0}>
```

Comment:
has no effect with CSS overrides

```suggestion
      <AppBar position="static" elevation={0}>
```